crawlers/d-crawler/dub.sdl

```diff
@@ -0,0 +1,9 @@
+name "crawler-demo"
```

Comment:
I'd be very hesitant to throw this out, since the parts of this that are finished _do_ work and are (somewhat) well tested; the issue is just that it's overengineered, overcomplicated, and very incomplete.

That said, we're not using this code (d-crawler) at all.